recipes/trilinos/recipe/12.10-fixes.patch

```diff
@@ -0,0 +1,323 @@
+diff -Naur 12.10/packages/PyTrilinos/cmake/PyTrilinosMakeTest.cmake 12.12/packages/PyTrilinos/cmake/PyTrilinosMakeTest.cmake
```

Comment:
Is this needed since this recipe doesn't build pytrilinos?

Reply:
I thought so, because it contains bug fixes, but after reviewing it, it looks like the fixes are all in the PyTrilinos package.